config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-ec2.yaml

```diff
@@ -62,8 +62,9 @@ periodics:
     test.kops.k8s.io/k8s_version: stable
     test.kops.k8s.io/kops_channel: alpha
     test.kops.k8s.io/networking: amazonvpc
-    testgrid-dashboards: kops-misc, sig-cluster-lifecycle-kops, sig-scalability-aws
+    testgrid-dashboards: kops-misc, sig-cluster-lifecycle-kops, sig-scalability-aws, sig-release-master-informing
```

Comment:
I see that this test has been failing since November 21. At this time, I don't think we should add that test to the master-informing board. It's going to confuse Release Team members and Release Managers. Similar, we shouldn't send alerts to `release-team@kubernetes.io` at this time.